### ISSUE TITLE
Project release

### ISSUE THREAD

**Issue by tunghoang290780** [mrc-donato/CHIPKIT-Tutorial#2]
Hi,

Do you plan to release the project and slides used in the MICRO tutorial?

Thanks

**Comment by mrc-donato** [comment]
Yes, we will release the project and the tutorial slides in the coming weeks. Stay tuned!

**Comment by xiejin03** [comment]
Hi, I'm an engineer from DJI SOC team, leading a custom ISP develop group. 
Paul Whatmough recommended me to look at this amazing tutorial. It really inspired me a lot. And I've got several questions:
1. on slice 16, what does DCO mean? and what's the purpose of the "Noise Gen." module?
2. will you open source the VGEN scripts? I think it is a great tool.
3. on slice 44, there is an ahb_slave_intf and an ahb_master_intf. The only difference is some signals are missing in aha_master_intf. And you already defined the source and sink mod port in each interface. So it seems duplicated to me that two ahb_intf are defined. Is there any reason to define two interfaces?
4. about the DNN engine, does it use the OOHLS method?

Thanks,
Jin

**Comment by mrc-donato** [comment]
Hi Jin,

Thanks for checking out the tutorial! We are working on the release of the material (including the VGEN scripts). In the meantime, here's an answer to your questions:

1. DCO stands for Digitally Controlled Oscillator and it is used to generate high-performance clock signals on-chip. The "Noise Gen." module is used to generate supply voltage noise for testing purposes. We are going to release both implementations with CHIPKIT. You can find more details in [this paper](https://ieeexplore.ieee.org/abstract/document/8387436)

3. The AHB protocol provides a definition not only between the master and the slave interfaces, but also interfaces with the decoder and the multiplexor. You can find an example of this configuration on slide 39. Defining two distinct interface modules helps organizing the connections between master, slaves, and the remaining interface blocks. 

4. We have not used high-level synthesis for designing the DNN Engine, but we are planning to add resources on the HLS Design flow in the future. We give a brief overview on slides 114-118
